src/stylesheets/components/_navigation.scss

```diff
@@ -0,0 +1,50 @@
+.app-c-navigation {
+  $navigation-height: 53px;
+  padding-right: $govuk-spacing-scale-3;
+  padding-left: $govuk-spacing-scale-3;
+  background-color: $app-light-grey;
+  font-family: $govuk-font-stack;
+  @include font-smoothing;
+  @include govuk-bold-19;
+
+  @include mq($until: desktop) {
+    display: none;
+  }
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    & > li {
+      box-sizing: border-box;
+      height: $navigation-height;
+      float: left;
+      line-height: $navigation-height;
+      -moz-box-sizing: border-box;
+      -webkit-box-sizing: border-box;
+
+      &:hover {
+        border-bottom: 4px solid $govuk-light-blue;
+      }
+
+      &.app-c-navigation--current-page {
+        border-bottom: 4px solid $govuk-blue !important;
+      }
+
+      a {
+        display: block;
+        padding: 0 20px;
+        color: $govuk-blue;
```

Comment:
Semantically it makes sense to use `$govuk-link-colour` and `$govuk-link-hover-colour` here? If we decided to change the link colour to a different colour in the palette we'd want it to change here too.